qiskit/optimization/problems/problem_type.py

```diff
@@ -34,6 +38,7 @@ class ProblemType(object):
     for LP, QP, and QCP or the topic titled Discrete Optimization
     for MILP, FIXEDMILP, NODELP, NODEQP, MIQCP, NODEQCP.
     """
+    # pylint: disable=invalid-name
     LP = CPXPROB_LP
```

Comment:
Just a question for my understanding: Why do we wrap the CPLEX enums here? Would it not be possible to forward this to a CPLEX class or inherit from one?

Reply:
I don't know. It is Jukub's original design. Maybe we can change it.

Reply:
I will think how it should be.